js/index.js

```diff
@@ -1 +1,61 @@
 // Your code goes here
+
+const navLinks = document.querySelectorAll('.nav-link');
+navLinks[0].addEventListener('click', (e) => e.preventDefault())
+navLinks[1].addEventListener('click', (e) => e.preventDefault())
+navLinks[2].addEventListener('click', (e) => e.preventDefault())
+navLinks[3].addEventListener('click', (e) => e.preventDefault())
```

Comment:
How could you write this as a loop to save yourself from repeating the same code?